bciers/apps/reporting/src/app/(authenticated)/authenticated/facilities/page.tsx

```diff
@@ -0,0 +1,37 @@
+"use client"
+
+import React, { useState, useEffect } from 'react';
+import { actionHandler } from "@/app/utils/actions";
+
+const Page: React.FC = () => {
+  const [message, setMessage] = useState<string>('');
```

Comment:
At the risk of having blinking pages initially before we have all the `<Suspense> ... </ >` straightened out, we shouldn't create dynamic states and unnecessary effects when we don't need them.
I'd suggest following the pattern used in the registration app for now, like [here](https://github.com/bcgov/cas-registration/blob/23b68db8fe1c86a450619f62e1010b8dceb9deb8/client/app/components/operations/Operation.tsx#L172)

Benefit is also to not have a bunch of async methods firing everywhere, but to have the page data loading happening in a more controlled way (and rendered server-side as much as possible).